schemas/url.yml

```diff
@@ -166,14 +166,15 @@
       type: keyword
       short: Query string of the request.
       description: >
-        The query field describes the query string of the request,
-        such as "q=elasticsearch".
+        The field contains the entire query string, excluding the leading `?`
+        character, such as "q=elasticsearch".
 
         The `?` is excluded from the query string. If a URL contains
```

Comment:
@johnmschoonover  If you exclude `?` in upper sentence, we should remove this repetition here